### ISSUE TITLE
Swiftlint is hanging for a long period of time before starting

### ISSUE THREAD

**Issue by mozeryansky** [realm/SwiftLint#2827]
### New Issue Checklist

- [X] Updated SwiftLint to the latest version
- [X] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

I upgraded my MacOS to 10.14.6, Xcode to 10.3, and Swiftlint to 0.34.0 (from 0.33.0). I'm not if Xcode or my MacOS is relevant but I wanted to mention. Since upgrading swiftlint, there is a significant time to just start the program. Doing `time swiftlint version` reports a real time of 10 seconds. Running `time swiftlint lint` on an empty folder reports a real time of ~22secs, where it used to be ~2secs. I think 2 seconds on an empty folder is quite slow, but 22 seconds is unusable, especially as a build phase in xcode.

What can I provide to help diagnose this issue?

### Environment

* SwiftLint version: 0.34.0
* Installation method: Homebrew
* Paste your configuration file: none used

**Comment by mozeryansky** [comment]
I just tried `time swiftlint lint --no-cache` and it said 16seconds, and a second run of `time swiftlint` now says 0.05secs. What's going on here?

**Comment by jpsim** [comment]
Very odd, I'm also on macOS 10.14.6 with Xcode 10.3, but I'm seeing 20-70 milliseconds for the scenarios you described:

```
$ mkdir gh2827
$ cd gh2827
$ time swiftlint version
0.34.0
swiftlint version  0.01s user 0.01s system 81% cpu 0.023 total
$ time swiftlint lint
Linting Swift files at paths
No lintable files found at paths: ''
swiftlint lint  0.03s user 0.02s system 71% cpu 0.068 total
$ xcode-select -p
/Applications/Xcode-10.3.app/Contents/Developer
```

If you clone this project and build it from source, do you get similar performance?

```
$ git clone https://github.com/realm/SwiftLint.git
$ cd SwiftLint
$ make install
```

**Comment by mozeryansky** [comment]
Right after filing this I commented that running `--no-cache` fixed the issue. I can run with or without `--no-cache` now, and it works as expected.

**Comment by jpsim** [comment]
So there’s no way for you to reproduce the problem again?

**Comment by mozeryansky** [comment]
I can't reproduce it anymore, no. Close? What caused t

**Comment by jpsim** [comment]
> What caused the problem?

That's a great question. I'm hesitant to close this, but there's not much point in keeping it open if this won't ever become actionable.

If you're comfortable, you could email me the contents of your cache which may share insights into how this happened. `jp [at] jpsim [dot] com`

**Comment by mozeryansky** [comment]
It's all work related code. The 0.33.0 version says "cache.json" and is 19.2 MB. And the 0.34.0 version looks "[sha].plist" and is 147 KB. Version 0.33.0 was slow in general for me, must have been loading and parsing the file. If it happens again, I'll look into sending it to you.

**Comment by jpsim** [comment]
Ok, since this currently isn't actionable I'll close the ticket, but am happy to dig further into it if any further info surfaces.

**Comment by AnthonyMillerSF** [comment]
This is happening sporadically for me now also. Can't seem to reliably reproduce this, but when it happens, it's pretty frustrating...
